src/containers/ConceptPage/ConceptForm/FormFooter.tsx

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright (c) 2021-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import React from 'react';
+import { injectT, tType } from '@ndla/i18n';
+import { useFormikContext } from 'formik';
+import EditorFooter from '../../../components/SlateEditor/EditorFooter';
+import SaveButton from '../../../components/SaveButton';
+import Field from '../../../components/Field';
+import { Article, PossibleStatuses, Values } from '../../../components/SlateEditor/editorTypes';
+import { FormikAlertModalWrapper, formClasses, FormikActionButton } from '../../FormikForm';
+import { ConceptType } from '../../../interfaces';
+
+interface Props {
+  entityStatus: { current: string };
+  inModal: boolean;
+  formIsDirty: boolean;
+  savedToServer: boolean;
+  isNewlyCreated: boolean;
+  showSimpleFooter: boolean;
+  onClose: () => void;
+  onContinue: () => void;
+  handleSubmit: (values: unknown) => void;
+  createMessage: (o: { translationKey: string; severity: string }) => void;
```

Comment:
hva er o? objekt? :)

Reply:
Vet ikke, egentlig, Originalt så var det bare PropTypes.func, så jeg bare ga den bokstaven o for objekt ja.